automations/request_screenshot/README.md

```diff
@@ -0,0 +1,16 @@
+# Request Screenshot
+If the PR lacks an image file, or link to an image in the description, apply a `no-screenshot` label and post a comment to request a screenshot. If the PR author updates the description, gitStream will remove the label.
+
+![Request Screenshot](request-screenshot.png)
+
+Conditions (all must be true):
+* The PR description lacks an image or link to an image.
+
```

Comment:
maybe say something about adding addtional conditions that will make sense for your team, for when to show this. For example, in our repo it make sense only when changing files under the `docs` dir